cmd/version.go

```diff
@@ -0,0 +1,24 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package cmd provides access to the embedded version of the binaries.
+// Taken from https://github.com/AgentZombie/go-embed-version.
+package cmd
```

Comment:
Is there other place to put it? Putting it under cmd package seems a little weird, as it's the top level dir for the 3 binaries.

Reply:
That was the suggested location from that link. It kinds of does feel like the right place to me, since it's a version that only applies to the binaries, all of which are under cmd/.

Would you rather something like cmd/version/?

Reply:
makes sense to me now